net/arp.lisp

```diff
@@ -130,7 +130,7 @@ Returns NIL if there is no entry currently in the cache, this will trigger a loo
   nil)
 
 (defun arp-expiration ()
-  (let ((time (1+ (get-internal-real-time))))
+  (let ((time (1+ (get-universal-time))))
```

Comment:
Is it an option to use internal-real-time throughout arp.lisp instead?  Is get-universal-time monotonic, or can it change backwards when setting the system date/time (in which case it should not be used for ARP timeouts).